data/apps/org.telegram.messenger.web.json

```
@@ -14,7 +14,7 @@
     ],
     "icon": "https://telegram.org/img/apple-touch-icon.png",
     "description": {
-        "en": "Telegram is a open-source messanger.",
-        "de": "Telegram ist ein quelloffener Messenger."
+        "en": "Telegram is a messanger where the client is open source.",
+        "de": "Telegram ist ein Messenger, bei dem der Client quelloffen ist."
```

```suggestion
        "en": "Telegram is a cloud-based mobile and desktop messaging app with a focus on security and speed.",
        "de": "Telegram ist eine Cloud-basierte mobile und Desktop-Messaging-App mit Schwerpunkt auf Sicherheit und Geschwindigkeit."
```

from the https://telegram.org website